client/main.lua

```diff
@@ -157,12 +174,12 @@ AddEventHandler('prison:client:Leave', function()
 	end
 end)
 
-RegisterNetEvent('prison:client:UnjailPerson')
+RegisterNetEvent('prison:client:UnjailPerson') ---- We need someone to help write the code to return the clothes to the player after getting out of prison ---
```

Comment:
Do not comment in PR's, do it in the github. You should remove it and request it in the comments of the PR.